pyfar/io/io.py

```diff
@@ -297,3 +228,210 @@ def write(filename, compress=False, **objs):
 
     with open(filename, 'wb') as f:
         f.write(zip_buffer.getvalue())
+
+
+def read_audio(filename, dtype='float64', **kwargs):
+    """
+    Import an audio file as :py:class:`~pyfar.classes.audio.Signal` object.
+
+    Parameters
+    ----------
+    filename : string, Path
+        Input file.
+    dtype : {'float64', 'float32', 'int32', 'int16'}, optional
+        Data type of the returned signal, by default ``'float64'``.
+        Floating point audio data is typically in the range from
+        ``-1.0`` to ``1.0``.  Integer data is in the range from
+        ``-2**15`` to ``2**15-1`` for ``'int16'`` and from ``-2**31`` to
+        ``2**31-1`` for ``'int32'``.
+    **kwargs
+        Other keyword arguments to be passed to ``soundfile.read``. This is
+        needed, e.g, to read RAW audio files.
+
+
+    Returns
+    -------
+    signal : Signal
+        :py:class:`~pyfar.classes.audio.Signal` object containing the audio
+        data.
+
+    Notes
+    -----
+    * This function is based on ``soundfile.read``.
+    * Reading int values from a float file will *not* scale the data to
+      [-1.0, 1.0). If the file contains ``np.array([42.6], dtype='float32')``,
+      you will read ``np.array([43], dtype='int32')`` for ``dtype='int32'``.
```

Comment:
Maybe we can omit this, if we are more explicit above.

Reply:
I'd suggest to keep it as an illustrative example